src/generic/srchctlg.cpp

```diff
@@ -506,15 +515,19 @@ void wxSearchCtrl::LayoutControls()
         searchMargin = 0;
         cancelMargin = 0;
     }
-    wxCoord textWidth = width - sizeSearch.x - sizeCancel.x - searchMargin - cancelMargin - FromDIP(1);
+
+    textWidth -= sizeSearch.x + sizeCancel.x + searchMargin + cancelMargin - FromDIP(1);
```

Comment:
Careful examination shows that the new code adds 2px to `textWidth` because the sign before `FromDIP(1)` hasn't been changed, although `= ... - FromDIP(1)` has been turned into `-= ... - FromDIP(1)`. Was this really intentional? I assume it wasn't...

Reply:
> Was this really intentional? I assume it wasn't...

It was not intentional at all, I simply missed that.
Thank you for noticing.